<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Start</class>
 <widget class="QMainWindow" name="Start">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1136</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>721</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Fire Services / Number of Staffed Fire In-Service Vehicle Hours per Capita (Urban and Rural Areas)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fire Services / Number of Residential Structure Fires with Losses per 1000 Households (Urban and Rural Areas)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fire Services / Station Notification Response Time (Urban and Rural Areas)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fire Services / Fire Operating Cost per Staffed In-Service Vehicle Hour (Urban and Rural Areas)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fire Services / Number of Injuries and Fatalities as a Result of Fires (Urban and Rural Areas)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Library Services / Annual Number of Library Service Hours per Capita</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Library Services / Number of Library Holdings per Capita</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Library Services / Library Use per Person (Electronic and Non-Electronic)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Library Services / Average Number of Time in Year Circulating Items are Borrowed</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Library Services / Library Operating Cost per Use</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Parks Services / All Parkland in Municipality as a Percent of Total Area of Municipality</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Parks Services / Amount of Parkland Available Per Resident (Natural, Maintained, Total)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Parks Services / Operating Cost per Hectare (Maintained and Natural Parkland Combined)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Police Services / Number of Total Police Staff (Officers and Civilians) per 100,000 Population</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Police Services / Reported Number of Total (Non-Traffic) Criminal Code Incidents per 100,000 Population</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Police Services / Total Crime Severity Index</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Police Services / Reported Number of Violent - Criminal Code Incidents per 100,000 Population</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Police Services / Violent Crime Severity Index</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Police Services / Clearance Rate - Violent Crime</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Police Services / Number of Criminal Code Incidents (Non-Traffic) per Police Officer</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Waste Management Services / Tonnes of all Material Collected per Household (Residential)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Waste Management Services / Operating Costs for Garbage Collection per Tonne (Residential)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Waste Management Services / Tonnes of Solid Waste Disposed per Household (Residential)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Waste Management Services / Operating Costs for Solid Waste Disposal per Tonne (All Streams)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Waste Management Services / Tonnes Solid Waste Divereted per Household (Residential)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Waste Management Services / Operating Costs for Solid Waste Disposal per Tonne (Residential)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Waste Management Services / Percentage of Solid Waste Diverted (Residential)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Water Services / Megalitres of Treated Water per 100,000 Population</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Water Services / Average Age of Infrastructure and Population Density of Serviced Community</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Water Services / Number of Water Main Breaks per 100Km of Water Distribution Pipe (excluding Service Connections and Hydrant Leads)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Water Services / Operating Cost for the Distribution/Transmission of Drinking Water per Km of Water Distribution Pipe</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Water Services / Operating Cost for the Treatment of Drinking Water per Megalitre of Drinking Water Treated</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Table</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>280</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Bar Graph</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>280</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Create LIne Graph</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
